.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: java
 
 before_install:
+  - if [[ "${TRAVIS_CPU_ARCH}" == "arm64" ]]; then
```

Comment:
Why do we need to install maven from apt-get?

Reply:
Especially given that there is a Maven Wrapper in the project that should be unused instead

Reply:
I have used the maven wrapper present in this.